Make build-debs failing due to missing build dir

## Description

A short summary of the issue.

## Steps to Reproduce
On debian 12. Just run `make build-debs`

## Expected Behavior
Starts building.

## Actual Behavior
```
~/securedrop-client$ poetry run make build-debs
Building SecureDrop Client Debian packages...
Script started, output log file is 'build/securedrop-client-20240508.log'.
script: cannot open build/securedrop-client-20240508.log: No such file or directory
                                                                                   Script done.
make: *** [Makefile:9: build-debs] Error 1
```

## Comments

`mkdir build` solves it.

--- a/Makefile
+++ b/Makefile
@@ -4,6 +4,9 @@ all: help
 .PHONY: build-debs
 build-debs: OUT:=build/securedrop-client-$(shell date +%Y%m%d).log
 build-debs: ## Build Debian packages
+	# "build-debs.sh" will create this directory, but we need it to exist
+	# before we call "script".
+	@mkdir -p build
 	@echo "Building SecureDrop Client Debian packages..."
 	@export TERM=dumb
 	@script \

